Description: We're not building from a git checkout, so DownloadPinnedClang() cannot work.
Author: Olivier Tilloy <olivier.tilloy@canonical.com>
Author: Nathan Pratta Teodosio <nathan.teodosio@canonical.com>
Forwarded: not-needed

--- a/tools/clang/scripts/build.py
+++ b/tools/clang/scripts/build.py
@@ -696,7 +696,6 @@ def main():
     cc = args.host_cc
     cxx = args.host_cxx
   else:
-    DownloadPinnedClang()
     if sys.platform == 'win32':
       cc = os.path.join(PINNED_CLANG_DIR, 'bin', 'clang-cl.exe')
       cxx = os.path.join(PINNED_CLANG_DIR, 'bin', 'clang-cl.exe')
@@ -707,8 +706,8 @@ def main():
       cxx = cxx.replace('\\', '/')
       lld = lld.replace('\\', '/')
     else:
-      cc = os.path.join(PINNED_CLANG_DIR, 'bin', 'clang')
-      cxx = os.path.join(PINNED_CLANG_DIR, 'bin', 'clang++')
+      cc = '/usr/bin/gcc'
+      cxx = '/usr/bin/g++'
 
     if sys.platform.startswith('linux'):
       MaybeDownloadHostGcc(args)
